app/hackathon_variables.py

```diff
@@ -40,8 +40,10 @@
                    'the opening ceremony will be at 7:00 pm.'
 
 # (OPTIONAL) When to arrive at the hackathon
-HACKATHON_LEAVE = 'Closing ceremony will be held on Sunday October 15th from 3:00 PM to 5:00 PM. ' \
-                  'However the projects demo fair will be held in the morning from 10:30 AM to 1 PM.'
+HACKATHON_LEAVE_HACKER_SPONSOR = 'Closing ceremony will be held on Sunday October 15th from 3:00 PM to 5:00 PM. ' \
+                         'However the projects demo fair will be held in the morning from 10:30 AM to 1 PM.'
+HACKATHON_LEAVE_OTHER = 'Closing ceremony will be held on Sunday October 15th from 3:00 PM to 5:00 PM. '
```

Comment:
I still don't know why you need 2 strings. Mentors may also want to know when is the demo fair.

Reply:
It was decided like this by HackUPC, I can put the same in the second one so we don't modify it in here

Reply:
Can we just leave it as one. Adds complexity and little to no value. Redirect here anyone from HackUPC who thinks otherwise...